backend/src/main/java/com/allog/dallog/category/domain/Category.java

```diff
@@ -0,0 +1,57 @@
+package com.allog.dallog.category.domain;
+
+import com.allog.dallog.category.exception.InvalidCategoryException;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "categories")
+public class Category {
```

Comment:
```suggestion
@Table(name = "categories")
@Entity
public class Category {
```

제 생각에는 Category 클래스를 가장 잘 나타내는 어노테이션은 `@Entity` 인 것 같은데, 위와 같이 변경하는 것은 어떨까요?

Reply:
저도 그게 맞는 것 같아서 고민하다가 일단 `Schedule`에 달려있는 순서로 적었네요. 수정하겠습니다~ 내일 리버한테도 알려주세요.

Reply:
아하 그렇군요! 감사합니다!